content/influxdb/v2.0/reference/cli/influx/v1/auth/create.md

```diff
@@ -18,31 +18,33 @@ influx v1 auth create [flags]
-|      | `--password`      | Password to set on the authorization                                                                   |             |                         |
-|      | `--read-bucket`   | Bucket ID to assign read permissions to                                                                |             |                         |
-|      | `--skip-verify`   | Skip TLS certificate verification                                                                      |             |                         |
-| `-t` | `--token`         | Authentication token                                                                                   | string      | `$INFLUX_TOKEN`         |
-|      | `--username`      | (Required) Token username                                                                              | string      |                         |
-|      | `--write-bucket`  | Bucket ID(s) to assign write permissions to                                                            | stringArray |                         |
+| Flag |                   | Description                                                                                                | Input type  | {{< cli/mapped >}}      |
+|:-----|:------------------|:-------------------------------------------------------------------------------------------------------    |:-----------:|:------------------------|
+| `-c` | `--active-config` | Config name to use for command                                                                             | string      | `$INFLUX_ACTIVE_CONFIG` |
+|      | `--configs-path`  | Path to the influx CLI configurations (default: `~/.influxdbv2/configs`)                                   | string      | `$INFLUX_CONFIGS_PATH`  |
+| `-d` | `--description`   | Token description                                                                                          | string      |                         |
+| `-h` | `--help`          | Help for the `create` command                                                                              |             |                         |
+|      | `--hide-headers`  | Hide the table headers (default: `false`)                                                                  |             | `$INFLUX_HIDE_HEADERS`  |
+|      | `--host`          | HTTP address of InfluxDB                                                                                   | string      | `$INFLUX_HOST`          |
+|      | `--json`          | Output data as JSON (default: `false`)                                                                     |             | `$INFLUX_OUTPUT_JSON`   |
+|      | `--no-password`   | Don't prompt for a password (to use the token, you must set a password with `influx v1 auth set-password`) |             |                         |
```

Comment:
Does this mean if you use this option (no password), you must set a password with `influx v1 auth set-password`but otherwise, you don't need to?

Reply:
Correct. If you don't include this flag, you are prompted to provide a password.